index.html

```diff
@@ -31,7 +33,7 @@ <h3 class="detail">I'm a software developer! I can help you build a product , fe
             <li><a href="#"> <img src="Vector.svg"class="socialmedia"/></a></li>
           </ul>
         </div>
-    </section>
+    </section></div>
     
 
 <section class="project">
```

Comment:
- [ ] Kindly consider centering your project cards on the page
- [ ] Kindly make sure the `background color, border,typo(font-size, font-weight)` on the tech stack matches with the one provided in Figma.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193918307-50288286-a5eb-4a21-b661-79e4c0f332b2.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193918485-17e4abd6-07f2-4f84-9037-80e721108461.png)